beacon_node/execution_layer/src/test_utils/execution_block_generator.rs

```diff
@@ -643,30 +646,51 @@ impl<T: EthSpec> ExecutionBlockGenerator<T> {
     }
 }
 
-pub fn generate_random_blobs<T: EthSpec, R: Rng>(
+pub fn load_test_blobs_bundle<E: EthSpec>() -> Result<(KzgCommitment, KzgProof, Blob<E>), String> {
+    let blob_bundle_bytes = match E::spec_name() {
+        EthSpecId::Mainnet => TEST_BLOB_BUNDLE_MAINNET,
+        EthSpecId::Minimal => TEST_BLOB_BUNDLE_MINIMAL,
+        EthSpecId::Gnosis => {
+            return Err("Test blobs bundle not available for Gnosis preset".to_string())
+        }
+    };
+
+    let BlobsBundle {
+        commitments,
+        proofs,
+        blobs,
+    } = BlobsBundle::<E>::from_ssz_bytes(blob_bundle_bytes)
+        .map_err(|e| format!("Unable to decode SSZ: {:?}", e))?;
+
+    Ok((
+        commitments
+            .get(0)
+            .cloned()
+            .ok_or("commitment missing in test bundle")?,
+        proofs
+            .get(0)
+            .cloned()
+            .ok_or("proof missing in test bundle")?,
+        blobs.get(0).cloned().ok_or("blob missing in test bundle")?,
+    ))
+}
+
+pub fn generate_blobs<E: EthSpec>(
     n_blobs: usize,
-    kzg: &Kzg<T::Kzg>,
-    rng: &mut R,
-) -> Result<(BlobsBundle<T>, Transactions<T>), String> {
-    let mut bundle = BlobsBundle::<T>::default();
-    let mut transactions = vec![];
-    for blob_index in 0..n_blobs {
-        let random_valid_sidecar = BlobSidecar::<T>::random_valid(rng, kzg)?;
+) -> Result<(BlobsBundle<E>, Transactions<E>), String> {
+    let (kzg_commitment, kzg_proof, blob) = load_test_blobs_bundle::<E>()?;
```

Comment:
Maybe we can we add a `verify_kzg_proof` test for the saved bundle to make sure we catch a change in the trusted setup?

Reply:
yeah that's a great idea - I'll add a test for that.

Reply:
Add in https://github.com/sigp/lighthouse/pull/4829/commits/d0a4b1d34e51ad143a7f9f75ade4977b4aed3164